src/game.cpp

```diff
@@ -11820,10 +11820,15 @@ void game::place_player( const tripoint &dest_loc )
         }
     }
 
-    if( veh1 != nullptr && veh1->part_with_feature(vpart1, "CONTROLS") >= 0 && u.in_vehicle ) {
-        add_msg(_("There are vehicle controls here."));
-        add_msg(m_info, _("%s to drive."),
+    if( veh1 != nullptr && veh1->part_with_feature( vpart1, "CONTROLS" ) >= 0 && u.in_vehicle) {
+        add_msg( _( "There are vehicle controls here." ) );
+        if (!veh1->is_furniture) {
```

Comment:
You should nearly never have a negated condition if you have a then *and* an else branch. Remove the negation and switch the then and else branches around. This applies to other places here as well.